test/cases/api.sh

```diff
@@ -185,7 +202,14 @@ function createReqFileAWS() {
   "customizations": {
     "packages": [
       "postgresql"
-    ]
+    ],
+    "subscription": {
+      "organization": ${API_TEST_SUBSCRIPTION_ORG_ID:-},
+      "activation-key": "${API_TEST_SUBSCRIPTION_ACTIVATION_KEY:-}",
+      "base-url": "http://cdn.redhat.com/",
```

Comment:
Is that an `http://` or an `https://` URL ?  IMO should be https

Reply:
Thanks! Updated. But I've no idea why http also works in gcp...Perhaps both can work? Anyway the default url is indeed https.